cpp/open3d/t/io/file_format/FilePLY.cpp

```diff
@@ -315,15 +399,21 @@ bool WritePointCloudToPLY(const std::string &filename,
                         num_points, it.first, it.second.GetLength());
                 return false;
             }
-        } else if (it.second.GetShape() != core::SizeVector({num_points, 1})) {
-            utility::LogWarning(
-                    "Write PLY failed. PointCloud contains {} attribute which "
-                    "is not supported by PLY IO. Only points, normals, colors "
-                    "and attributes with shape (num_points, 1) are supported. "
-                    "Expected shape: {} but got {}.",
-                    it.first, core::SizeVector({num_points, 1}).ToString(),
-                    it.second.GetShape().ToString());
-            return false;
+        } else {
+            auto shape = it.second.GetShape();
+            if (shape.size() < 2 || shape[0] != num_points) {
```

Comment:
What is the purpose of this 2 here? Maybe add some documentation of use constant variables

Reply:
In the shape-check block for extra attributes, I check that the attribute tensor has at least two dimensions (i.e., (`num_points`, `channels`)), ensuring that we only support attributes with a defined channel dimension. I added comments to clarify this requirement.